app/android/.gitignore

```diff
@@ -0,0 +1,11 @@
+gradle-wrapper.jar
```

Comment:
Could this be moved to the main .gitignore (one already in the repo before this branch), and still work?